.github/workflows/ci.yml

```diff
@@ -0,0 +1,101 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - develop
+
+    tags:
+      - '*'
+
+  pull_request:
+
+env:
+  MAIN_REPO: Oceans-1876/challenger-portal
```

Comment:
What is `MAIN_REPO` used for?

Reply:
We can remove this if we want for now. My thought was when we are ready to push images somewhere, I can use this to differentiate whether the image should be pushed to a registry based on whether it's built on github or built on local using act. For example, I can add a line:

`if: github.repository == env.MAIN_REPO
      uses: elgohr/Publish-Docker-Github-Action@3.04`

If that were run using act, github.repository won't evaluate.